tianshou/trainer/offpolicy.py

```diff
@@ -24,6 +24,9 @@ def offpolicy_trainer(
     test_fn: Optional[Callable[[int, Optional[int]], None]] = None,
     stop_fn: Optional[Callable[[float], bool]] = None,
     save_fn: Optional[Callable[[BasePolicy], None]] = None,
+    save_checkpoint_fn: Optional[Callable[[int, int, int], None]] = None,
```

Comment:
1. What is the differences between save_fn and save_checkpoint_fn?
2. I think we add too much code just to support "resume training". More code means harder to read/rewrite. I don't know whether this single method worth this.

Reply:
save_fn saves the best policy, while save_checkpoint_fn saves periodically 

but this issue is required by many users (even one years ago! https://discuss.pytorch.org/t/is-there-a-way-to-resume-logging-checkpoint-while-using-tensorboard/82307)